share/spack/gitlab/cloud_pipelines/stacks/e4s-oneapi/spack.yaml

```diff
@@ -269,6 +269,11 @@ spack:
     pipeline-gen:
     - build-job:
         image: ecpe4s/ubuntu20.04-runner-x86_64-oneapi:2023-01-01
+    - any-job:
```

Comment:
This should probably only be applied to `build-job` configs. And in general I would recommend things that are related be grouped in a list of lists.

```yaml
  - build-job:
        before_script:
        - - . /bootstrap/runner/view/lmod/lmod/init/bash
          - module use /opt/intel/oneapi/modulefiles
          - module load compiler
```